src/libs/dutil/gdiputil.cpp

```diff
@@ -210,3 +210,55 @@ HRESULT DAPI GdipHresultFromStatus(
 
     return E_UNEXPECTED;
 }
+
+/********************************************************************
+GetDpiForMonitor - returns the X and Y DPI values for specified monitor.
+
+********************************************************************/
+BOOL DAPI GetDpiForMonitor(
```

Comment:
This doesn't really belong in GdipUtil. How about a new DpiUtil? Make this `DpiGetForWindow`?